MultiplayerMod/Multiplayer/Commands/Tools/Build.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using MultiplayerMod.Core.Unity;
+using MultiplayerMod.Game.Building;
+using MultiplayerMod.Game.Events.Tools;
+using UnityEngine;
+
+namespace MultiplayerMod.Multiplayer.Commands.Tools;
+
+[Serializable]
+public class Build : IMultiplayerCommand {
+
+    private BuildEventArgs @event;
```

Comment:
What is a meaning of `@` in the var name?

Reply:
> The @ character prefixes a code element that the compiler is to interpret as an identifier rather than a C# keyword.

From [C# reference](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/verbatim). In this case the `event` keyword is used as a variable name.

Reply:
Thanks for the explanation.
But I still concerned and don't really see a strong reason to use keyword for variable names here (and above for `override`) and getting to deal with this mechanism to override it.
Here for instance we can use `args`, `buildEvent` or even `buildEventArgs`.

Reply:
This thing has another causality, it's not like
> use a keyword for variable name

It's the name happens to be a keyword in C#, and that's good the language allows to use keywords as identifiers.
Anyway, I'll think once more, at first sight `event` is the most suitable name in the command domain.

Reply:
Decided to change it to `arguments` since this data is actually command arguments🤔.